docs/conf.py

```diff
@@ -0,0 +1,55 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'meilisearch-python'
+copyright = '2022, Charlotte Vermandel'
```

Comment:
@alallema 2019 should be enough legally (disclaimer, I'm not a lawyer :smile:), but I'm not sure if you would rather do 2019-2022 to match the license file. You would need to remember to update both places each year if so.

```suggestion
copyright = '2019, Meili SAS'
```

Reply:
Changed it

Reply:
Yes thank you so much @sanders41 you right and thank you for the review ❤️